resources/public/api/conf/1.0/business_create.yaml

```diff
@@ -18,7 +18,7 @@ post:
     - $ref: './common/headers.yaml#/components/parameters/correlationId'
 
   security:
-    - User-Restricted:
+    - Application-Restricted:
         - write:self-assessment
```

Comment:
Not sure on the scope to use for each endpoint, I have left it the same as when the endpoints were user restricted. I assume it is still the same?

Reply:
I believe we use "read" scope for endpoints that only retrieve data (e.g. list endpoint). 
If an endpoint posts, amends or deletes data, then it needs a "write" scope.